app/views/partials/trials-list.html

```diff
@@ -158,11 +158,11 @@ <h4 title="{{ trial.title }}">{{ trial.title }}</h4>
               </dl>
             </li>
             {% endif %}
-            {% if trial.sponsors.length > 0 %}
+            {% if trial.fundedBy.length > 0 %}
             <li>
               <dl class="sponsors">
                 <dt>Sponsored by</dt>
-                <dd>{{ joinListOfNames(trial.sponsors) | safe }}</dd>
+                <dd>{{ joinListOfNames(trial.fundedBy) | safe }}</dd>
```

Comment:
This was changed somewhere else? (I mean using `trial.fundedBy` instead of `trial.sponsors`)